From d52821e5a3ba2ea37cac1a0beb0f2c1bed91d492 Mon Sep 17 00:00:00 2001
From: Gabor Igloi <gabor.igloi@citrix.com>
Date: Fri, 20 Oct 2017 17:31:56 +0100
Subject: [PATCH 01/13] Update dependencies for new cohttp

cohttp v0.99.0 has been split into multiple OPAM packages, and the
ocamlfind packages have also been renamed.

Signed-off-by: Gabor Igloi <gabor.igloi@citrix.com>
Signed-off-by: Marcello Seri <marcello.seri@citrix.com>
---
 message-switch-core.opam | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/message-switch-core.opam b/message-switch-core.opam
index c3296b4..12af7c4 100644
--- a/message-switch-core.opam
+++ b/message-switch-core.opam
@@ -13,7 +13,8 @@ build: [
 depends: [
   "jbuilder" {build}
   "ocamlfind" {build}
-  "cohttp" {>= "0.21.1" & < "0.22.0"}
+  "base-unix"
+  "cohttp" {>= "0.21.1"}
   "rpc" {>= "1.9.51"}
   "sexplib"
   "ppx_sexp_conv"
-- 
2.14.1

